jsonp/src/main/webapp/index.html

```diff
@@ -16,8 +16,18 @@
 -->
 <!DOCTYPE html>
 <!-- Plain HTML page that kicks us into the app -->
-<html>
+<html lang="en">
 <head>
-<meta http-equiv="Refresh" content="0; URL=jsonp.jsf">
+    <meta charset="UTF-8">
+    <title>jsonp</title>
 </head>
-</html>
+<body>
+<div style="text-align:center">
+
+    <h1>Hello There! Welcome to WildFly!</h1>
+    <h2>The jsonp application has been deployed and running successfully.</h2>
+    <a href="http://localhost:8080/jsonp/jsonp.jsf" title="Go to JSONP Form">Access the jsonp application here</a>
```

Comment:
I believe the href can be just `jsonp.jsf` since customers can use different host names and ports.

Reply:
Changed. Thanks!